package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "test": "jest",
+    "test": "jest --verbose",
```

Comment:
Making this verbose only blows up CI logs. Any reason you've included it here? As long as tests pass, we basically don't care about them :wink:

Reply:
Oh sorry, i used this for debugging purposes and forgot it.